server/db/db.js

```diff
@@ -663,17 +663,37 @@ async function getInstallationWithSessionId(sessionId, clientParam) {
   return null
 }
 
-async function getHistoricAlertsByAlertApiKey(alertApiKey, maxHistoricAlerts, maxTimeAgoInMillis, clientParam) {
-  let client = clientParam
-  const transactionMode = typeof client !== 'undefined'
-
+async function getActiveAlertsByAlertApiKey(alertApiKey, maxTimeAgoInMillis, clientParam) {
   try {
-    if (!transactionMode) {
-      client = await pool.connect()
-    }
+    const results = await helpers.runQuery(
+      'getActiveAlertsByAlertApiKey',
+      `
+      SELECT s.id, s.state, b.unit, s.num_presses, i.incident_categories, s.created_at
+      FROM sessions AS s
+      LEFT JOIN buttons AS b ON s.button_id = b.button_id
+      LEFT JOIN installations AS i ON s.installation_id = i.id
+      WHERE i.alert_api_key = $1
+      AND (
+        s.state <> $2
```

Comment:
the `<>` operator was confusing to me at first. I had to look it up - personally I think that `!=` would make the code easier to maintain, but I don't feel like it needs to be changed.

Reply:
You're right. I changed it. Thanks!